src/lib.rs

```diff
@@ -4,6 +4,26 @@
 #![feature(thread_local)]
 #![allow(unused_imports)]
 
+//! # RustPOSIX
+//! Welcome to the RustPOSIX microvisor. 
+//! This serves as the OS-in-a-process in the [Lind architecture](https://github.com/Lind-Project/lind-docs/tree/main). 
+//! ## Overview
+//!
+//! A microvisor is a piece of software that lives within the same process as the cages. It provides a POSIX interface, allowing cages to believe they have their own operating system.
+//!
+//! With this crate, you can:
+//! - Fork and execute other cages.
+//! - Read and write files on disk.
+//! - Communicate over the network.
+//! - Manage shared memory.
+//! 
+//! Note this is a re-design of the original (now obsolete) [SafePOSIX](https://github.com/Lind-Project/nacl_repy), which was created using the [RepyV2](https://github.com/SeattleTestbed/docs/blob/master/Programming/RepyV2Tutorial.md) sandbox.
+//! 
+//! ## Building RustPOSIX
```

Comment:
These sorts of things are usually not comments for Rustdoc (which is meant to be read by users of the library, not developers working on it.)

Reply:
will refactor